docs/installation.md

```diff
@@ -101,19 +99,21 @@ You may want to configure a few additional components of Lychee, such as:
 - Database
 - Session
 
+They are documented in the `.env.example` file, which may have been copied into your `.env`. They are also listed on the [configuration](configuration.html) page.
+
 ##### Advanced Configuration (Dangerous)
 
-Some advanced options cannot be configured through the `.env` file, but you may take a look into the directory `config/` to find such options.
-Usually, you don't have to change them and can go with the defaults.
+Some advanced options cannot be configured through the `.env` file. If you really need to change them, look in the `config/` directory.
 
 **If you don't know what they're doing, do not change them.**
 
 ## Web Server Configuration
 
 ### General
 
-Configure your web server's root to be the `public` directory.
-The `index.php` in this directory serves as the interface for all HTTP requests to Lychee's API.
+Configure your web server to make the `public` directory accessible via a URL. This could be the root of your web server (short and convenient) or, if you prefer, a subfolder such as `photos`. The `index.php` in this directory serves as the interface for all HTTP requests to Lychee's API.
```

Comment:
```suggestion
Configure your web server to make the `public` directory accessible via a URL. This could be the root of your web server (short and convenient) or, if you prefer, a subpath such as `photos`. The `index.php` in this directory serves as the interface for all HTTP requests to Lychee's API.
```

URLs don't have folders or directories. File systems do. Technically, the thing between two slashes is called component, but I believe nobody will understand that term. But "subfolder" really confuses me.

Moreover, I believe we should mention that the `AP_URL` variable in the `.env` file must match the URL.

Reply:
Thanks; I wasn't happy with the word "subfolder" when I was writing it but I couldn't come up with a better alternative at the time...